### PR TITLE
LPS-183542 | Add @ignore on some FDS tests that are failing

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
Add @ignore on some FDS tests that are failing.

> FDSViews#CanAssertPagination 
> FDSViews#CanUpdateNumberOfViews
> FDSViews#CanUseSpecialCharactersInNameField
> FDSEntry#WithSpecialCharacters

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-183542

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:stop

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:stop:frontend

**Comment by abhnerramos** [comment]
hey @mateusralv , LGTM! Could you check it, please?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase [hunk 1]
```diff
@@ -289,8 +289,12 @@ definition {
 	}
 
 	@description = "LPS-179056. Confirm that the user can create a data set entry with special chatacters"
+	@ignore = "true"
 	@priority = 5
 	test WithSpecialCharacters {
+
+		// Ignored due to LPS-183512
+
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase:297]:
```suggestion

		

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase [hunk 2]
```diff
@@ -289,8 +289,12 @@ definition {
 	}
 
 	@description = "LPS-179056. Confirm that the user can create a data set entry with special chatacters"
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase:291]:
```suggestion
	@description = "Ignored due to LPS-183512 | LPS-179056. Confirm that the user can create a data set entry with special chatacters"
```

**Review by mateusralv — APPROVED** [review]
Hey @abhnerramos it looks good and could you remove the comments and add them in the description? Thanks :)

**Comment by abhnerramos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase (hunk 2 above)
Comment (abhnerramos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase]: Thanks for the example.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase (hunk 1 above)
Comment (abhnerramos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSEntry.testcase]: Will do that! Thanks for the feedback

**Comment by abhnerramos** [comment]
ci:forward

**Pull request closed without merging**